api-parent/access-api/src/main/java/com/sap/cloud/environment/servicebinding/api/ServiceBindingMerger.java

```diff
@@ -4,14 +4,21 @@
 
 package com.sap.cloud.environment.servicebinding.api;
 
+import javax.annotation.Nonnull;
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.List;
 
-import javax.annotation.Nonnull;
-
+/**
+ * A {@link ServiceBindingAccessor} that combines the result of multiple other {@link ServiceBindingAccessor}s. How the
+ * individual results will be combined (merged) can be configured.
+ */
 public class ServiceBindingMerger implements ServiceBindingAccessor
```

Comment:
This class is used to combine ("merge") the result of multiple `ServiceBindingAccessor` instances, so that one can consume services from different sources.
Therefore, I think it makes sense to have this class public since customers might want to create a customized `ServiceBindingAccessor` instances that should also consider multiple sources.